tests/run-make/symbol-mangling-hashed/rmake.rs

```diff
@@ -0,0 +1,264 @@
+// ignore-tidy-linelength
+//! Basic smoke test for the unstable option `-C symbol_mangling_version=hashed` which aims to
+//! replace full symbol mangling names based on hash digests to shorten symbol name lengths in
+//! dylibs for space savings.
```

Comment:
Improvement suggestions for this test comment are welcomed. The original Makefile had zero comments so this is already more context than no comments...